plugins/epos/plugin.py

```diff
@@ -301,16 +301,9 @@ def rda_a1_03m(self):
 
         points = 0
         msg = "Metadata can not be found"
-        try:
-            item_id_http = idutils.to_url(
-                self.item_id,
-                idutils.detect_identifier_schemes(self.item_id)[0],
-                url_scheme="http",
-            )
-            points, msg = ut.metadata_human_accessibility(self.metadata, item_id_http)
-            msg = _("%s \nMetadata found via Identifier" % msg)
-        except Exception as e:
-            logger.error(e)
+        if len(self.metadata) > 1:
```

Comment:
Since `self.metadata` is a pandas.DataFrame it is better to use such object's methods. A better approach would be:

`if not self.metadata.empty:`